### PR TITLE
Feature/update java selenium etc

### PR DESCRIPTION
**Pull request by ivy-lli**
Update to Java 11
Update Selenium
Update to Junit5
Update maven plugin versions
Use Selenide (Firefox) as driver provider

### CODE REVIEW

**Comment by ivy-lli** [comment]
Build: https://zugprojenkins/view/unstable-monitor/job/supplements-primeui-tester/job/feature%252FupdateJavaSeleniumEtc/

File: primeui-tester/pom.xml [hunk 1]
```diff
@@ -40,6 +40,8 @@
 
 	<properties>
 		<project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+    <junit.engine.version>5.5.2</junit.engine.version>
+    <maven.surefire.version>3.0.0-M3</maven.surefire.version>
```
Comment (ivy-dhu) [inline primeui-tester/pom.xml]: Correct alignment

File: primeui-tester/pom.xml [hunk 2]
```diff
@@ -66,33 +68,30 @@
 	</profiles>
 
 	<dependencies>
+	  <dependency>
+      <groupId>com.codeborne</groupId>
+      <artifactId>selenide</artifactId>
+      <version>5.5.0</version>
```
Comment (ivy-dhu) [inline primeui-tester/pom.xml]: alignment

File: primeui-tester/pom.xml [hunk 3]
```diff
@@ -66,33 +68,30 @@
...
-		<dependency>
-			<groupId>junit</groupId>
-			<artifactId>junit</artifactId>
-			<version>4.12</version>
-			<scope>test</scope>
-		</dependency>
+      <groupId>org.junit.jupiter</groupId>
+      <artifactId>junit-jupiter-engine</artifactId>
+      <version>${junit.engine.version}</version>
+    </dependency>
```
Comment (ivy-dhu) [inline primeui-tester/pom.xml]: alignment

**Pull request merged**